src/AAVE/Modal/SupplyModal.jsx

```diff
@@ -317,7 +431,7 @@ function depositErc20(amount) {
  * @returns
  */
 function getNewHealthFactor(chainId, address, asset, action, amount) {
-  const url = `https://aave-api.pages.dev/${chainId}/health/${address}`;
+  const url = `https://aave-data-service-7a85eea3aebe.herokuapp.com/${chainId}/health/${address}`;
```

Comment:
Please check all the files and replace the base URLs.

Reply:
Replaced